src/fdc3/dotnet/DesktopAgent/src/MorganStanley.ComposeUI.DesktopAgent/Infrastructure/Internal/Fdc3DesktopAgentMessageRouterService.cs

```diff
@@ -26,14 +26,13 @@
 using MorganStanley.ComposeUI.Fdc3.DesktopAgent.Converters;
 using MorganStanley.ComposeUI.Fdc3.DesktopAgent.DependencyInjection;
 using MorganStanley.ComposeUI.Fdc3.DesktopAgent.Exceptions;
-using MorganStanley.ComposeUI.Messaging;
-using MorganStanley.ComposeUI.Messaging.Abstractions;
+using MorganStanley.ComposeUI.MessagingAdapter.Abstractions;
 
 namespace MorganStanley.ComposeUI.Fdc3.DesktopAgent.Infrastructure.Internal;
 
 internal class Fdc3DesktopAgentMessageRouterService : IHostedService
```

Comment:
These services are not "MessageRouterService" anymore. I actually like that the services are independent of the underlying messaging implementation. My only concern is that the messaging adapter interface at the moment is copying the messagerouter interface, something we should iterate on to consider the ComposeUI needs directly rather than force adapting other messaging solutions with possibly different behaviors to the MessageRouter behavior.

Reply:
this should be still renamed!